calicoctl/commands/node/diags.go

```diff
@@ -22,10 +22,12 @@ import (
 	"path/filepath"
 	"strings"
 	"time"
+	"bufio"
+	"bytes"
 
 	"github.com/docopt/docopt-go"
 	log "github.com/sirupsen/logrus"
-	shutil "github.com/termie/go-shutil"
```

Comment:
Hm, I missed that, I agree with what you're saying, my IDE must have thought different.